holidayextras/countdown-notify-lambda
# TRIP-190: Improve logging

#### What does this PR do? (please provide any background)

Improves the logging so that we can better consume the data within sumologic

#### What tests does this PR have?

Unit tests covering the new logger.

#### How can this be tested?

Follow the instructions to setup the lambda function to run in staging and then run `npm start` Ensure you see the new style logging.

#### Any tech debt?

No

#### Screenshots / Screencast

#### What gif best describes how you feel about this work?
![](http://i.giphy.com/11AbJT85iWPirC.gif)

- I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---
